Source/devices/oni/AcqBoardONI.cpp

```diff
@@ -115,12 +115,10 @@ bool AcqBoardONI::detectBoard()
         {
             LOGC ("FTDI Library version: ", major, ".", minor, ".", patch);
         }
-        if (evalBoard->getFirmwareVersion (&major, &minor))
+        if (evalBoard->getFirmwareVersion (&major, &minor, &patch))
         {
-            LOGC ("Open Ephys ECP5-ONI FPGA open. Gateware version v", major, ".", minor);
-            //if (major >= 16) //For now we use this, we will use the proper versioning
-            //    hasBNO[0] = true;
-            hasI2cSupport = major >= 2;
+            LOGC ("Open Ephys ECP5-ONI FPGA open. Gateware version v", major, ".", minor + "." + patch);
+            hasI2cSupport = major >= 1 && minor > 5;
```

Comment:
This like this would fail with version 2.1 for example.
Although, thinking about it, I believe we should change it so this plugin only works with versions 0.x and 1.x
Version 2 is planned and will break API, So maybe if major > 2 it should report an error (and say to update the plugin) and return the same as board not detected., and then for 0.1 to 1.4 does not have i2c support, from 1.5 to 1.infinity, it does ;)